schemas/triniti/news/command/inspect-article-seo/1-0-0.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<pbj-schema xmlns="http://gdbots.io/pbj/xsd"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://gdbots.io/pbj/xsd http://gdbots.io/pbj/xsd/schema.xsd">
+
+  <schema id="pbj:triniti:news:command:inspect-article-seo:1-0-0">
+    <fields>
+      <field name="node_ref" type="node-ref" required="true"/>
+      <field name="providers" type="string" format="slug" rule="set">
+        <description>
+          Provides a list of providers to inspect article node ref.
```

Comment:
@digitalartcrew please update this description to be more along the lines of what you see for collect-article-stats